.github/workflows/main.yaml

```diff
@@ -121,6 +120,14 @@ jobs:
         test-registration-e2e,
         test-registration1-e2e,
       ]
+    # This wasn't wasn't being triggered if any tests were skipped
```

Comment:
I don't totally follow this comment. It seems like it refers to old code that you've since replaced? Maybe a comment more like "we run this job only if tests are successful or skipped, without this condition, [insert bad things] happen because [reasons]"

Reply:
That's a good point, edited the comment for clarity:
https://github.com/bcgov/cas-registration/pull/2201/commits/b01fa505e5383cbb56172527abc4ebc91e3c472d